spec/lite_spec_helper.rb

```diff
@@ -32,7 +32,7 @@
   begin
     require 'byebug'
   rescue LoadError
-    require 'ruby-debug'
+    require 'ruby-debug' rescue LoadError
```

Comment:
This isn't the correct syntax to rescue LoadError.

Reply:
Fixed